.github/workflows/ci.yml

```
@@ -20,7 +20,7 @@ jobs:
           bundler-cache: true
           ruby-version: 3.0
       - name: set up JDK 1.8
-        uses: actions/setup-java@v1
+        uses: actions/setup-java@v4
         with:
           java-version: 1.8
```

Old version string isn't supported https://github.com/actions/setup-java/blob/main/docs/switching-to-v2.md
Assuming this really old Java version is actually needed
```suggestion
          java-version: 8
```